src/server/enrollment-system.js

```diff
@@ -192,6 +192,7 @@ function veteranToSpouseInfo(veteran) {
       middleName: veteran.spouseFullName.middle,
       familyName: veteran.spouseFullName.last,
       suffix: veteran.spouseFullName.suffix,
+      relationship: 2,
```

Comment:
Just want to verify with Mehedi whether or not we should always pass 2 or if it should be based on the relationship status.